src/get-started/wordcount-example.md

```diff
@@ -890,24 +890,104 @@ PCollection<KV<String, Long>> wordCounts = windowedWords.apply(new WordCount.Cou
 # This feature is not yet available in the Beam SDK for Python.
 ```
 
-### Writing results to an unbounded sink
+## StreamingWordCount example
 
-When our input is unbounded, the same is true of our output `PCollection`. We
-need to make sure that we choose an appropriate, unbounded sink. Some output
-sinks support only bounded output, while others support both bounded and
-unbounded outputs. By using a `FilenamePolicy`, we can use `TextIO` to files
-that are partitioned by windows. We use a composite `PTransform` that uses such
-a policy internally to write a single sharded file per window.
+The StreamingWordCount example is a streaming pipeline that reads Pub/Sub
+messages from a Pub/Sub subscription or topic, and performs a frequency count on
+the words in each message. Similar to WindowedWordCount, this example applies
+fixed-time windowing, wherein each window represents a fixed time interval. The
+fixed window size for this example is 15 minutes. The pipeline outputs the
```

Comment:
It should read "15 seconds"
(source at `https://github.com/apache/beam/blob/master/sdks/python/apache_beam/examples/streaming_wordcount.py#L77`